master/internal/command/tensorboard_manager.go

```diff
@@ -45,8 +45,9 @@ const (
 type TensorboardRequest struct {
 	CommandParams
 
-	ExperimentIDs []int `json:"experiment_ids"`
-	TrialIDs      []int `json:"trial_ids"`
+	ExperimentIDs []int           `json:"experiment_ids"`
+	TrialIDs      []int           `json:"trial_ids"`
+	Context       archive.Archive `json:"context"`
```

Comment:
I just got added as a code owner (I usually don't review when this happens) but I noticed this and I just reviewed @hamidzr 's PR on this same code. This is now a shared object between the echo and grpc-gateway/grpc API. For the API to receive the context, it'll need to be deser'd and passed where this `TensorboardRequest` is made https://github.com/determined-ai/determined/blob/master/master/internal/command/tensorboard_manager.go#L175-L179 here and we probably should also support it in the new api?

This will be my only only/sorry if I'm commenting totally without context.

Reply:
thanks for this catch - I think I can piggyback off of the user_files directly instead of adding a context field here.